racunaljka.js

```diff
@@ -0,0 +1,54 @@
+function Body(length) {
+    this.length = length;
+}
+
+Body.prototype.growth = function(newLength) {
+    this.length += newLength;
+}
+
+function Head(left, right) {
+    this.leftEye = left;
+    this.rightEye = right;
+}
+
+Head.prototype.changeColor = function(color) {
```

Comment:
This method does not make sense, why for changing of a color you would provide another head? Just update the existing one:
```
Head.prototype.changeEyeColor = function(colorLeft, colorRight) {
 this.leftEye = colorLeft;
 this.rightEye = colorRight;
}
```